### ISSUE TITLE
How to call multiclass? Best loss function?

### ISSUE THREAD

**Issue by rhalherrero** [maxvfischer/keras-image-segmentation-loss-functions#2]
Great repo!
Could you please give an example of how to call one of your multiclass losses (e.g. multiclass_weighted_dice_loss)?

Have you found if one these multiclass loss functions works (in most cases) better than the others?

Thanks!
